History.md

```diff
@@ -0,0 +1,133 @@
+
+v3.0.0 / 2017-07-27
+===================
+
+  * Flush on first message (#113)
```

Comment:
Can we more diligent about the changelog here? My workflow is to typically start with git-changelog, and cleaning it up. e.g. here this could be (take my descriptions with a grain of salt, I'm not a js dev):

```
Version 3.0.0 (July 28th, 2017)
==============================
 * [New](https://github.com/segmentio/analytics-node/pull/110): Migrate to ES6. This is a breaking change. The `new keyword` is now required while initializing the client.
```
// Previously:
const a = Analytics()

// Now:
const a = new Analytics()
```

 * [Fix](https://github.com/segmentio/analytics-android/pull/112): Rename `flushAfter` setting to `flushInterval`. This is a breaking change.
 * [Fix](https://github.com/segmentio/analytics-node/pull/113): Flush when the client first receives a message.
* [Internal](link to commit): Remove caching of dependencies on Circle CI.
* [Internal](link to commit): Add License
...
```